source/features/releases-tab.tsx

```diff
@@ -17,7 +17,7 @@ import {appendBefore, highlightTab, unhighlightTab} from '../helpers/dom-utils';
 const getCacheKey = (): string => `releases-count:${getRepo()!.nameWithOwner}`;
 
 async function parseCountFromDom(): Promise<number> {
-	const moreReleasesCountElement = await elementReady('.file-navigation .octicon-tag + strong');
+	const moreReleasesCountElement = await elementReady('.Layout-sidebar .BorderGrid-cell h2 a[href$="/releases"] .Counter');
```

Comment:
I don’t think it needs to be so specific

Reply:
I took it from here

https://github.com/refined-github/refined-github/blob/9b60bfe4049e4f02a75d800784b9ce8784723394/source/features/clean-repo-sidebar.tsx#L10

Reply:
We can fix both

Reply:
Looks like we went though a few bugs with this selector already I will make a comment code https://github.com/refined-github/refined-github/pull/4428